README.md

```diff
@@ -1,60 +1,67 @@
 <div align="center">
     <div>
         <!-- Load images from raw.githubusercontent.com to enable image rendering when viewed from https://www.npmjs.com/package/@ni/eslint-config -->
-        <img src="https://raw.githubusercontent.com/ni/javascript-styleguide/HEAD/resources/logo.svg" alt="JavaScript, TypeScript, and NI logo" width="200px">
+        <img src="https://raw.githubusercontent.com/ni/javascript-styleguide/HEAD/resources/logo.svg" alt="JavaScript, TypeScript, and NI logo" width="300" height="100">
     </div>
 </div>
 
 # NI JavaScript and TypeScript Style Guide
+
+[![NPM Version](https://img.shields.io/npm/v/@ni/eslint-config.svg)](https://www.npmjs.com/package/@ni/eslint-config)
+
 Welcome to NI's JavaScript and TypeScript linter rules for [ESLint](https://eslint.org/docs/user-guide/getting-started).
 
 ## Installation
 
 Install `@ni/eslint-config` and its peer dependencies.
 
-Use [npm view](https://docs.npmjs.com/cli/view.html) to list the correct versions of each package to install yourself.
+Use [`npm view`](https://docs.npmjs.com/cli/view.html) to list the correct versions of each package to install yourself.
 
 ```bash
 npm view @ni/eslint-config peerDependencies
 ```
 
-Alternatively, use [install-peerdeps](https://www.npmjs.com/package/install-peerdeps) as a shortcut to install the packages for you.
+Alternatively, use [`npx install-peerdeps`](https://www.npmjs.com/package/install-peerdeps) as a shortcut to install the packages for you.
 
 ```bash
 npx install-peerdeps --dev @ni/eslint-config
 ```
 
-### Angular
+### Additional Angular-specific Installation
 
-See [instructions below](#usage-angular) to use a schematic to install dependencies.
+See [instructions below](#angular) to use a schematic to install dependencies.
 
 ## Usage
 
 ### JavaScript
 
 Extend `@ni` in the [ESLint configuration](https://eslint.org/docs/user-guide/configuring/configuration-files#configuration-file-formats).
 
-```json
+```js
 {
-    "extends": "@ni"
+    extends: '@ni'
 }
 ```
 
 ### TypeScript
 
-Extend `@ni/eslint-config/typescript` in the ESLint configuration. Configure the `@typescript-eslint` plugin and the project's TypeScript configuration.
+Extend `@ni/eslint-config/typescript` and `@ni/eslint-config/typescript-requiring-type-checking` (type checked rules) in the [ESLint configuration](https://eslint.org/docs/user-guide/configuring/configuration-files#configuration-file-formats).
 
-```json
+The type checked rules will also require the `parserOptions.project` configuration to be set to the project's TypeScript configuration. The type checked rules run a bit slower as they utilize the TypeScript compiler for type information.
```

Comment:
I think we can move information about rules that require type checking to the performance section below and reduce the word count here. This information doesn't have to be at the forefront as subsequent sections work just as well as reference links. The following combines concise instructions here and performance below. Again, users hate reading and want to flow through the setup fast.

"Extend `@ni/eslint-config/typescript` and `@ni/eslint-config/typescript-requiring-type-checking` in the [ESLint configuration](https://eslint.org/docs/user-guide/configuring/configuration-files#configuration-file-formats). Set the `parserOptions.project` configuration to the project's TypeScript configuration."

"
## Performance

## TypeScript

`@ni/eslint-config/typescript-requiring-type-checking` includes rules that require type checking that run slower as they utilize the TypeScript compiler for type information. If there are situations where the analysis time for enabling the type checked rules is an excessive burden you may consider creating a separate eslint configuration that avoids extending the type checked rules and omits the `parserOptions.project` configuration to run in specific scenarios. See discussion in the [performance section](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/TYPED_LINTING.md#performance) of the Getting Started - Linting with Type Information guide.

## Angular

Deviations ...
"

Reply:
I like it! thanks 👍